docs/scanner-quickstart.md

```diff
@@ -409,31 +402,41 @@ autoUpdate:
 
 ## Register Scan Node
 
-Your scan node has an Ethereum address that makes two main features possible:
+Your scan node has an Ethereum address that makes some features possible:
 
 - Receiving detection bots to run
-- Asserting an authority on the outputted alerts
+- Asserting authority on the outputted alerts
+- Identification for rewards and slashing
   
-While this address remains as the main identity, it must be owned by a different wallet. After registration, the scan node is minted as an NFT (ERC721) and transferred to this owner. You can check the [smart contract documentation here](smart-contracts.md).
+The private key for the scan node is generated at the `forta init` step. Please prefer continuing with this private key and do not replace it with your custom private key in order to avoid confusion and security risks.
+
+**Make sure you have set the `chainId` in your config.yml correctly before registering your node.** Your scan node can be registered only once and to scan a specific chain.
+
+You can register your node by
 
-In the future, the owner wallet will allow you to disable your scan node remotely and avoid slashing while you do maintenance (for a short period) or when you decide to shut down your node entirely. Right now, `forta disable` and `forta enable` commands are available to you to do the same using the scan node private key.
+- doing `forta authorize pool --id <your-pool-id>`,
+- copying the token over to Forta App as described in the [scanner pool management](scanner-pools.md) page of the docs.
 
-To register your node to the registry contract, you can run `forta register --owner-address <address>`. **Make sure you have set the `chainId` in your config.yml correctly before executing this.** Your scan node can be registered only once and to scan a specific chain. The owner wallet address needs to be a different address than your scan node address.
+Alternatively, you can
 
-!!! warning "Action requires funds"
-    You need to fund your scan node address with some Polygon (Mainnet) MATIC to be able to send this transaction. You can find out your scan node address with `forta account address`.
+- do `forta authorize pool --id <your-pool-id> --polygonscan`,
+- find out the `ScannerPoolRegistry` contract address from the [smart contract addresses page](smart-contracts.md),
```

Comment:
can we just provide a polygonscan link here that leads directly to the `registerScannerNode` method?

Reply:
We can't do that yet because we don't know the contract address yet.

Reply:
On a second thought, it would be good to do this because the contracts will be ready before we merge this.

Reply:
Done!